src/runtime/pytype.cs

```diff
@@ -111,7 +111,7 @@ internal IntPtr GetSlot(TypeSlotID slot)
         internal static TypeFlags GetFlags(BorrowedReference type)
         {
             Debug.Assert(TypeOffset.tp_flags > 0);
-            return (TypeFlags)Util.ReadCLong(type, TypeOffset.tp_flags);
+            return (TypeFlags)Util.ReadInt32(type, TypeOffset.tp_flags);
```

Comment:
This is not right. It's documented to be an `unsigned long`: https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_flags